.circleci/config.yml

```diff
@@ -377,6 +377,11 @@ jobs:
                 command: |
                     ./scripts/release/create-api-downloads-pr.sh mobile-maps-ios ${VERSION}
                 when: always
+            - run:
+                name: Update Package Manifest & Podspec
+                command: |
+                    ./scripts/release/update-spm-manifest.sh ${VERSION} "$(cat MapboxMaps.xcframework.zip.checksum)" 10.0.0-beta.8 10.0.0-beta.13
```

Comment:
I wonder if there's scope here for including the update script [this PR](https://github.com/mapbox/mapbox-maps-ios/pull/82) introduces.

Also, what are these versions here? MapboxCoreMaps & MapboxCommon?

Reply:
I'm going to make a change to use the Cartfile instead.

For your update_version script, I actually posted a comment wondering why you are updating podspec. I don't see why you are doing that.